plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestUriBasedS3SecurityMappingsProvider.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.s3;
+
+import com.sun.net.httpserver.HttpServer;
```

Comment:
This is an internal JDK API that we shouldn't use. See `TestJwkService` for how to use `TestingHttpClient` to test the code after switching to Airlift HTTP client.